format/binary.md

```diff
@@ -51,12 +51,14 @@ This document is based on:
 	- [Roblox Float Format](#roblox-float-format)
 
 ## Document Conventions
-This document assumes a basic understanding of Rust's conventions for numeric types. For example:
+We assume a basic understanding of Rust's conventions for numeric types. For example:
```

Comment:
I think that a document like this should avoid having pronouns of any kind, kinda like Wikipedia's writing style. We should keep this how it is or reword this to be passive somehow.